File: WebApp/Scripts/app/model/UR_DOC.js
Ext.define('WEBAPP.model.UR_DOC', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'DK', type: 'string' },
        { name: 'DN', type: 'string' },
        { name: 'DD', type: 'string' },
        { name: 'UK', type: 'string' },
        { name: 'CREATE_TIME', type: 'date' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'date' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' }
    ]
});

File: WebApp/Models/UR/UR_DOC.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class UR_DOC : JCLib.Mvc.BaseModel
    {
        public string DK { get; set; }
        public string DN { get; set; }
        public string DD { get; set; }
        public string UK { get; set; }
        public DateTime CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public DateTime UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
    }
}